### PR TITLE
add jquery-bgiframe@3.0.1 w/ git auto-update

### LINKED ISSUE #9890: [Request] Add bgiframe
**Library name:** bgiframe
**Git repository url:** https://github.com/brandonaaron/bgiframe
**License(s):** MIT

### PR DESCRIPTION
**Pull request by sufuf3**
close #9890, cc @brandonaaron
@kennynaoh please help me review this PR, thank you.

Pull request for issue: #9890 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/brandonaaron/bgiframe
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/jquery-bgiframe
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 156
   - Count of watchers: 13
   - Count of forks: 75
 * NPM download stats (optional):
   - Downloads in the last day: 0
   - Downloads in the last week: 25
   - Downloads in the last month: 103

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/jquery-bgiframe/3.0.1/jquery.bgiframe.min.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+!function(t){"function"==typeof define&&define.amd?define(["jquery"],t):"object"==typeof exports?module.exports=t:t(jQuery)}(function(t){function e(t){return t&&t.constructor===Number?t+"px":t}t.fn.bgiframe=function(i){if(i=t.extend({top:"auto",left:"auto",width:"auto",height:"auto",opacity:!0,src:"javascript:false;",conditional:/MSIE 6\.0/.test(navigator.userAgent)},i),!t.isFunction(i.conditional)){var o=i.conditional;i.conditional=function(){return o}}var n=t('<iframe class="bgiframe"frameborder="0"tabindex="-1"src="'+i.src+'"style="display:block;position:absolute;z-index:-1;"/>');return this.each(function(){var o=t(this);if(i.conditional(this)!==!1){var a=o.children("iframe.bgiframe"),r=0===a.length?n.clone():a;r.css({top:"auto"==i.top?(parseInt(o.css("borderTopWidth"),10)||0)*-1+"px":e(i.top),left:"auto"==i.left?(parseInt(o.css("borderLeftWidth"),10)||0)*-1+"px":e(i.left),width:"auto"==i.width?this.offsetWidth+"px":e(i.width),height:"auto"==i.height?this.offsetHeight+"px":e(i.height),opacity:i.opacity===!0?0:void 0}),0===a.length&&o.prepend(r)}})},t.fn.bgIframe=t.fn.bgiframe});
+//# sourceMappingURL=jquery.bgiframe.min.js.map
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-bgiframe/3.0.1/jquery.bgiframe.min.js]: Where is your map file?

**Comment by sufuf3** [comment]
I added the map file. @kennynaoh please help me review this PR, thank you.

**Comment by kennynaoh** [comment]
@sufuf3 Please fill in the detail information of comment table. Thanks!

**Comment by sufuf3** [comment]
@kennynaoh I updated the comment table. Please help me review this PR, thank you.

**Comment by sufuf3** [comment]
@kennynaoh I changed to git auto-update. Please help me review this PR, thank you.

**Comment by kennynaoh** [comment]
@sufuf3 Please also remember to update the comment title next time. Thanks!

**Comment by kennynaoh** [comment]
LGTM, ping @cdnjs/intern2

**Comment by PeterDaveHello** [comment]
@kennynaoh `s/comment title/PR title/` ...

**Pull request merged**